src/strategies/jwt.bearer.strategy.js

```diff
@@ -103,6 +110,17 @@ const verify = (jwt_payload, done) => {
         }
         if (isUser) {
             context['isUser'] = isUser;
+            // Test that required fields are populated
+            let validInput = true;
+            requiredJWTFields.forEach((field) => {
+                if (!jwt_payload[field]) {
```

Comment:
There is a warning `Generic Object Injection Sink`. 
```suggestion
                if (!jwt_payload[`${field}`]) {
```